src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -253,13 +251,13 @@ export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
                 data-testid={`edit-cql-library-button-${selectedCQLLibrary.id}-edit`}
               >
                 {/* edit and version: must be draft and have ownership, else view only*/}
-                {selectedCQLLibrary.createdBy === userName &&
+                {checkUserCanEdit(selectedCQLLibrary.createdBy, []) &&
```

Comment:
can we do it like you are doing it in EditCqlLibrary.tsx? calling this hook once and assigning it to a variable

Reply:
@adongare Thanks a lot Ashok, just made the change.